src/core/sys/posix/time.d

```diff
@@ -126,8 +126,6 @@ else version (Solaris)
 else version (CRuntime_Bionic)
 {
     enum CLOCK_MONOTONIC        = 1;
-    enum CLOCK_MONOTONIC_RAW    = 4;
-    enum CLOCK_MONOTONIC_COARSE = 6;
```

Comment:
OK, I didn't notice the recent changes to `core.time` to use those.  I had these constants inside `version(CRuntime*)` checks, as opposed to `version(linux)`, because they are also defined by glibc and bionic, and I give precedence to the C runtime in those cases.

But I don't really care, easier to collapse them.  I'll do that.